todos-backend/src/app.dbService.ts

```diff
@@ -0,0 +1,42 @@
+import { Injectable } from '@nestjs/common';
+import { DatabaseService } from './database/database.service';
+
+@Injectable()
+export class AppDBService {
```

Comment:
add error handling throughout, calls to the db can fail, i they fail the controller should return response 500